posthog/models/filters/mixins/funnel.py

```diff
@@ -71,6 +75,42 @@ def microseconds_from_days(days):
         return microseconds * FunnelWindowDaysMixin.milliseconds_from_days(days)
 
 
+class FunnelWindowMixin(BaseParamMixin):
```

Comment:
I think this replaces `FunnelWindowDaysMixin`, so we should get rid of it in this PR? Ensures we can do a clean sweep & update things everywhere. And the default in `base.py` means its backwards compatible, until the frontend is upgraded too.

We don't have to worry about cache issues on updating, since the keys will be different.

Reply:
the mixin handles the parameter being parsed correctly when they're passed from the frontend. If i remove it now, funnel_window_days will never be parsed

Reply:
.. Which is fine I think, if we stop using it? (Might mean adding the default to trends as well, but that should be it?)

Reply:
Ah, I see, FUNNEL_WINDOW_INTERVAL takes the value from FUNNEL_WINDOW_DAYS right now.

So, two step process. 👍